### DISCUSSION TITLE
python E501 line too large

### DISCUSSION THREAD

**Discussion by csfreebird** [syl20bnr/spacemacs#15731] [Q&A] [answered]
After upgrading emacs from 27.1 to 28.1, python warning 'E501, ling too large ...' cannot be disabled.
Anyone ran into my problem?

my layer configuration about python :

```
     lsp
     (python :variables
             python-backend 'lsp
             ;; python-lsp-server 'mspyls
             python-fill-column 200
             python-formatter 'yapf
             python-sort-imports-on-save t)
```

**Comment by pataquets** [comment, marked as answer]
Syntax checking under Spacemacs is done using [flycheck](https://github.com/flycheck/flycheck), which relies in language specific external checkers.
As per [Python layer's docs](https://develop.spacemacs.org/layers/+lang/python/README.html#syntax-checking), it's `flake8` who is tasked with doing this for Python.

Check out [language-specific syntax checkers documentation](https://www.flycheck.org/en/latest/user/syntax-checkers.html#configure-syntax-checkers) to learn how to configure yours to your liking.
